plugins/push/frontend/public/templates/mobile-message-preview.html

```diff
@@ -16,10 +16,28 @@
         </div>
         <img v-if="media" class="cly-vue-push-notification-mobile-preview__ios-media" v-bind:src="media" /> 
         <div class="cly-vue-push-notification-mobile-preview__ios-title">
-            {{title}}
+            <template v-if="isHTML">
+                <template v-for="(component) in titlePreviewComponentsList">
+                    <keep-alive>
```

Comment:
why keep-alive?

Reply:
A lot of times the dynamic components are re-rendered (on each input key plus whenever the localizations are changed). I ended up following the Vue recommendations and instead use <keep-alive> as a means of cache so that fo the same component on each render we use the previous instance instead of creating a new one.